openid_connect.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "swd", ">= 1.0.0"
   s.add_runtime_dependency "webfinger", ">= 1.0.1"
   s.add_runtime_dependency "rack-oauth2", ">= 1.6.1"
+  s.add_runtime_dependency "openssl", ">= 2.2.0"
```

Comment:
Is `openssl v2.2.0+` really needed?
This requirements would cause gem version incompatibility for many developers.

Reply:
Any prior openssl is fine

Reply:
OK, I'll remove this line after merged.
Thanks!